packages/devtools-proxy-support/src/system-ca.ts

```diff
@@ -58,12 +107,14 @@ export async function systemCA(
   existingOptions: {
     ca?: NodeJSCAOption;
     tlsCAFile?: string | null | undefined;
-  } = {}
+  } = {},
+  allowCertificatesWithoutIssuer?: boolean // defaults to false
```

Comment:
Do we want to also expose this in connect method options or is the idea that we'd allow to activate it with the env var?

Reply:
If we wanted to expose it the idea is that we'd forward it all the way to the user in the UI, right?

I am pretty hopeful that this won't be necessary, and it's a temporary solution anyway. So yeah, the environment variable would be the way to get back to the current (i.e. "broken" 1.44) behavior if somebody does end up needing that for some reason

Reply:
Ohhhhh, wait, I totally misread the variable name and thought it's the other way around, okay yeah, not exposing it at all makes sense 😄